src/domexpression.js

```diff
@@ -159,6 +159,39 @@ var _ = Mavo.DOMExpression = $.Class({
 					return "";
 				}
 
+				if (env.value instanceof Promise) {
+					env.value
+						.then(value => {
+							env.value = Mavo.Primitive.format(value, {
+								attribute: this.attribute,
+								element: this.element
+							});
+
+							this.output(env.value);
+
+							Mavo.hooks.run("domexpression-update-end", env);
+						})
+						.catch(error => {
+							env.expr.error(`A promise in the expression [${env.expr.expression}] was rejected with the reason`, error);
+
+							// Return empty string if a promise is rejected
+							env.value = Mavo.Primitive.format("", {
+								attribute: this.attribute,
+								element: this.element
+							});
```

Comment:
Should we print the reason why the promise has been rejected? Or is it enough to write it to the console?

Can we simply write `env.value = ""` in that case?